tox.ini

```diff
@@ -1,27 +1,25 @@
 [tox]
-envlist = {py36,py37,py38,py39}-{cov,nocov,diffcov}, qa, docs
+envlist = qa, docs, py{36,37,38,39}-{nocov,cov,diffcov}
 skip_missing_interpreters = True
 
 [testenv]
 commands =
-    python -c "import sys; print(sys.version); print(sys.executable)"
-    python -c "import os,pprint; os.makedirs('_dynamic', exist_ok=True); pprint.pprint(dict(os.environ), stream=open('_dynamic/ENV."{envname}"', 'wt'))"
+    python housekeep.py setup
     nocov: python -m nose2 -v {posargs}
     cov: python -m coverage run -m nose2
     cov: python -m coverage combine
     cov: python -m coverage html --directory=htmlcov/{envname}
     cov: python -m coverage report -m --fail-under=100
-    diffcov: python -m coverage xml
+    cov: python -m coverage xml
```

Comment:
XML generation should be part of the `cov` step. This simplifies deps, making full tox faster (and using less disk space)